x-pack/plugins/security_solution/public/resolver/view/panel.tsx

```diff
@@ -3,191 +3,245 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import React, { memo, useCallback, useMemo, useContext } from 'react';
-import {
-  EuiPanel,
-  EuiBadge,
-  EuiBasicTableColumn,
-  EuiTitle,
-  EuiHorizontalRule,
-  EuiInMemoryTable,
-} from '@elastic/eui';
-import euiVars from '@elastic/eui/dist/eui_theme_light.json';
+
+import React, { memo, useCallback, useMemo, useContext, useLayoutEffect, useState } from 'react';
 import { useSelector } from 'react-redux';
-import { i18n } from '@kbn/i18n';
-import { SideEffectContext } from './side_effect_context';
-import { ResolverEvent } from '../../../common/endpoint/types';
-import * as event from '../../../common/endpoint/models/event';
-import { useResolverDispatch } from './use_resolver_dispatch';
+import { useHistory } from 'react-router-dom';
+// eslint-disable-next-line import/no-nodejs-modules
```

Comment:
@XavierM how did we get around this in the original PR? Should we consider removing this lint rule?